### PR TITLE
disable testing and pushing to quay.io for CentOS7 image 

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

**Comment by zmiklank** [comment]
[test-all]

File: imagestreams/redis-centos.json [hunk 1]
```diff
@@ -9,59 +9,6 @@
...
-          "tags": "redis"
-        },
-        "from": {
-          "kind": "ImageStreamTag",
-          "name": "5-el8"
-        },
-        "referencePolicy": {
-          "type": "Local"
-        },
-        "name": "latest"
```
Comment (phracek) [inline imagestreams/redis-centos.json:27]: The latest part should be there and changed to something that is still supported. like 6-el8.

File: imagestreams/redis-centos.json [hunk 2]
```diff
@@ -9,59 +9,6 @@
...
-        "annotations": {
-          "description": "Provides a Redis database on CentOS. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/redis-container/tree/master/5/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Redis available on OpenShift, including major version updates.",
-          "iconClass": "icon-redis",
-          "openshift.io/display-name": "Redis (Latest)",
-          "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "tags": "redis"
-        },
-        "from": {
-          "kind": "ImageStreamTag",
-          "name": "5-el8"
```
Comment (phracek) [inline imagestreams/redis-centos.json:22]: `6-el8` should be marked as 'latests'

**Review by phracek — CHANGES_REQUESTED** [review]
Only one thing. The latest tag should be still present in redis-centos.json file.

Please update also main README.md file and mark CentOS 7 as deprecated or delete it.

**Comment by zmiklank** [comment]
> Only one thing. The latest tag should be still present in redis-centos.json file.

This is now updated. Can you recheck?

> Please update also main README.md file and mark CentOS 7 as deprecated or delete it.

THank for catching this. I have updated readme for redis 7. As for other readmes; this PR is based on the nginx one (https://github.com/sclorg/nginx-container/pull/256). The READMEs should be kept as they are, as we still have the centos7 images available on quay.io. We will just stop doing their updates and rebuilds.

**Review by phracek — CHANGES_REQUESTED** [review]
Only two nitpicks. CentOS 7 is mentioned in main README.md file still as an available image on quay.io

And CentOS 7 is mentioned as supported in the list of supported versions.
I would also prefer in example on main README.md file to replace CentOS 7 with CentOS Stream 8.

**Comment by zmiklank** [comment]
> Only one nitpicks.
> 
> And CentOS 7 is mentioned as supported in the list of supported versions. I would also prefer in example on main README.md file to replace CentOS 7 with CentOS Stream 8.

@pkubatrh, could you please share you opinion on this?

**Comment by phracek** [comment]
> 

Like @pkubatrh mentioned in nginx-container. Let's keep it.

**Review by phracek — APPROVED** [review]

**Comment by phracek** [comment]
[test-all]

**Pull request merged**